code/script.js

```diff
@@ -1,51 +1,316 @@
+  You can also go for 3 or 5 years.<br>
+  Please fill in, if you want 1, 3 or 5 years`, "bot");
+
+  // add input field
+  document.getElementById("input-wrapper").innerHTML = `
+  <form id="plan-form">
+  <label for="plan-input">Plan</label>
+  <input id="plan-input" type="text" />
+  <button class="send-btn" id="send-button-plannchoice">
+  Send
+  </button>
+  </form>`;
+  // submit
+  document.getElementById("plan-form").addEventListener("submit", planSubmit);
+  // not "click" 
+}
+
+function planSubmit(event) {
+  // connect to form!
+  event.preventDefault();
+  // console.log(document.getElementById("domain-input").value);
+  planValidation();
+}
+
+const planValidation = () => {
+  plan = document.getElementById("plan-input").value;
+  console.log(plan);
+  switch (plan) {
+    case "1":
+      plan = "1 year";
+      break;
+    case "3":
+      plan = "3 years";
+      break;
+    case "5":
+      plan = "5 years";
+      break
+    default:
+      plan = "invalid";
+  }
+  if (plan == "invalid") {
+    showMessage("Invalid input, try again.", "bot");
+    setTimeout(planSelection, 1000);
+  } else {
+    setTimeout(planConfirmation, 1000);
+  }
+}
+
+const planConfirmation = () => {
+  showMessage(`I pick: <br>${plan}`, "user");
+  calculation();
+}
+
+//3. calculation of price
+const calculation = () => {
+  let planDuration = document.getElementById("plan-input").value;
+  console.log(planDuration);
+  sum = ((FeeDomain + FeeHosting) * planDuration);
+  sum = sum * (1 - discount);
+  // console.log(sum);
+  setTimeout(invoiceMessage, 1000);
+}
+
+const invoiceMessage = () => {
+  showMessage(`Your final annual price is:<br>${sum} credits per year.`, "bot");
+  setTimeout(offerDeveloper, 1000);
 }
+// 4. offer help thru developer
+const offerDeveloper = () => {
+  showMessage(`
+  Do you need some help creating your website?<br>
+  We have commited junior developers from Technigo at hand.
+  `, "bot");
 
-// Set up your eventlisteners here
+  // create 2 buttons
+  document.getElementById("input-wrapper").innerHTML = `
+<div id="assistance-wrapper" style="display: flex">
+  <button id="yes" value="yes">YES</button>
+  <button id="no" value="no">No</button>
+</div>`;
+  // additional div for targeting content and serving as flex container, input-wrapper needs to ramain
+  // do not nest ""
 
-// When website loaded, chatbot asks first question.
-// normally we would invoke a function like this:
-// greeting()
-// But if we want to add a little delay to it, we can wrap it in a setTimeout:
-// setTimeout(functionName, timeToWaitInMilliSeconds)
-// This means the greeting function will be called one second after the website is loaded.
+  const assistanceWrapper = document.getElementById("assistance-wrapper");
+
+  assistanceWrapper.addEventListener("click", function (event) {
+    // do not target predefined input-wrapper for every query! target needs to be different, therefore add additional div in costum html
```

Comment:
you really perservered with solving this problem! good job with this, you found a way that works for your project and went with it